style.css

```diff
@@ -0,0 +1,163 @@
+body{
+    margin: 0;
+    padding: 0;
+
+    font-family: sans-serif;
+}
+.container{
+    position: relative;
+
+    width: 100%;
+    height: 100vh;
+
+    overflow: hidden;
+}
+.container .slider{
+    position: absolute;
+    top: 0;
+    left: 0;
+
+    width: 500%;
+    height: 100%;
+
+    animation: animate 10s linear infinite;
+    animation-delay: 2s;
+}
+.container .slider .slide{
+    position: relative;
+    float: left;
+
+    width: 20%;
+    height: 100%;
+}
+.container .slider .slide .caption{
+    position: absolute;
+    bottom: 60px;
+    left: 60px;
+    right: 60px;
+
+    padding: 30px;
+
+    background: rgba(0,0,0,0.5);
+    box-sizing: border-box;
+
+    transition: 0.5s;
+}
+.container .slider .slide .caption h2{
+    margin: 0 0 20px;
+    padding: 0;
+
+    color: #fff;
+    font-size: 48px;
+}
+.container .slider .slide .caption p{
+    margin: 0;
+    padding: 0;
+
+    color: #fff;
+    font-size: 18px;
+}
+.container .slider .slide.slide1{
+    background: url(./images/image1.jpeg);
+    background: url(./images/image1.jpeg);
+    background-size: cover;
+    background-position: center;
+}
+.container .slider .slide.slide2{
+    background: url(./images/image2.webp);
+    background: url(./images/image2.webp);
+    background-size: cover;
+    background-position: center;
+}
+.container .slider .slide.slide3{
+    background: url(./images/image3.webp);
+    background: url(./images/image3.webp);
+    background-size: cover;
+    background-position: center;
+}
+.container .slider .slide.slide4{
+    background: url(./images/image4.png);
+    background: url(./images/image4.png);
+    background-size: cover;
+    background-position: center;
+}
+.container .slider .slide.slide5{
+    background: url(./images/image5.webp);
+    background: url(./images/image5.webp);
+    background-size: cover;
+    background-position: center;
+}
+@keyframes animate
+{
+    0%
+    {
+        left: 0;
+    }
+    20%
+    {
+        left: -50%
+    }
```

Comment:
Your First pic isn't synchorizing with the animation as compared to all other pics, can you please tweak around the settings here so that we will get a uniform movement / animation ?

Reply:
i think here we need to chenge keyframe at start for that...& i allredy used delay at start so may thats why it happens
?

Reply:
Well all of your pics are set at certrain% value for 10% of the animation, but for the first picture you didn't set that. You have used -50% and then directly jumped to 100%, so I guess that image isn't getting the same effect as the others.

Reply:
i just added new changes please check it out..

Reply:
exactly i just notice it and try to resolve it as u told me to do that bhai..